src/nvim/api/vim.c

```diff
@@ -985,9 +985,10 @@ Buffer nvim_create_buf(Boolean listed, Boolean scratch, Error *err)
 ///            as a "\r", not as a "\n". |textlock| applies. It is possible
 ///            to call |nvim_chan_send()| directly in the callback however.
 ///                 ["input", term, bufnr, data]
+///          - newline_conv: (boolean, default false) Convert "\n" to "\r\n".
```

Comment:
https://github.com/neovim/neovim/issues/14557#issuecomment-1366194970 mentions that the default should be `true`. Why is it false in this PR?

Reply:
see this comment https://github.com/neovim/neovim/pull/26384#discussion_r1413784935

Reply:
I think we should try it. It was already broken, this likely improves more than it breaks. 

@zeertzjq any strong objection?